app/scripts/angular-apimock.js

```diff
@@ -120,6 +135,8 @@ angular.module('apiMock', [])
 						key +
 						(topLevel ? '' : ']'));
 					});
+				} else if (toSerialize === undefined || toSerialize === '') {
+					parts.push(encodeUriQuery(prefix));
```

Comment:
Because the result is still useful. The whole point of supporting query parameters is that it changes the `.json` file you get in the end. If I'd ignore empty query values then we'd still return the base `.json`. See https://github.com/seriema/angular-apimock/blob/chore-44-raise-test-coverage/test/spec/services/angular-apimock.js#L731-L746

Or see the bugfix it was related to. (see if you can find the reference from the commit, see it as a challenge ;) )